tests/cases/fourslash/codeFixCorrectReturnValue28.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path='fourslash.ts' />
+
+// @strict: true
+
+// @Filename: test.ts
+//// (function() {
+////   const config = {
+////     values: [],
+////     value: {},
+////   };
+////
+////   config.values.push(config.value);
+//// })();
```

Comment:
```suggestion
//// }();
```
The test, as it currently is, doesn't reproduce the crash. However, the code in the original report, or enclosing the parens like `(function () { /* ... */ }());`, will

Reply:
Updated